### PR TITLE
Improve docs for Standalone GC

### PR DESCRIPTION
**Pull request by talSofer**
Closes https://github.com/treeverse/product/issues/610

### CODE REVIEW

File: docs/howto/garbage-collection/standalone-gc.md [hunk 1]
```diff
@@ -18,66 +18,58 @@ experimental
...
+{% include toc_2-3.html %}
 
-## About
+## What is Standalone GC? 
 
-Standalone GC is a limited version of the Spark-backed GC that runs without any external dependencies, as a standalone docker image.
+Standalone GC is a simplified version of the Spark-backed GC that runs without any external dependencies, delivered as a standalone
+docker image. It supports S3 and [self-managed S3 compatible storages](#s3-compatible-clients) such as MinIO.    
 
 ## Limitations
```
Comment (yonipeleg33) [inline docs/howto/garbage-collection/standalone-gc.md:33]: I would reference the lab tests from here or the "What is Standalone GC?" section. I think it is important to inform the reader about the limited tests performed.

File: docs/howto/garbage-collection/standalone-gc.md [hunk 2]
```diff
@@ -18,66 +18,58 @@ experimental
...
 
 {: .note .warning }
-> Standalone GC is experimental and offers limited capabilities compared to the [Spark-backed GC]({% link howto/garbage-collection/gc.md %}). Read through the [limitations](./standalone-gc.md#limitations) carefully before using it.
+> Standalone GC is experimental and offers limited capabilities compared to the [Spark-backed GC]({% link howto/garbage-collection/gc.md %}). For large scale environments, we recommend using the Spark-backed solution.
 
-{% include toc_2-4.html %}
+{% include toc_2-3.html %}
 
-## About
+## What is Standalone GC?
```
Comment (yonipeleg33) [inline docs/howto/garbage-collection/standalone-gc.md:28]: Very nit (extra space):
```suggestion
## What is Standalone GC?
```

**Review by ozkatz — APPROVED** [review]
Thank you!

**Review by yonipeleg33 — CHANGES_REQUESTED** [review]
Thanks, looking good overall!
Had some comments

File: docs/howto/garbage-collection/standalone-gc.md [hunk 3]
```diff
@@ -18,66 +18,58 @@ experimental
...
 docker pull treeverse/lakefs-sgc:<tag>
 ```
 
-## Usage
+## Setup
 
 ### Permissions
-To run `lakefs-sgc`, you'll need AWS and LakeFS users, with the following permissions:
+
+To run `lakefs-sgc`, you'll need AWS and lakeFS users, with the following permissions:
```
[the remark this answers is not in the document]
Comment (talSofer) [reply docs/howto/garbage-collection/standalone-gc.md]: done, let me know what you think

File: docs/howto/garbage-collection/standalone-gc.md [hunk 4]
```diff
@@ -288,9 +289,10 @@ In this prefix, you'll find 2 objects:
 
 To delete the objects marked by the GC, you'll need to read the `deleted.csv` file, and manually delete each address from AWS.
 
-It is recommended to move all the marked objects to a different bucket instead of deleting them directly.
+We recommend to start by backing up the marked objects on a different bucket before deleting them. Then, deleting them directly from 
+the backup location. 
 
-Here's an example bash script to perform this operation:
+Here's an example bash script to perform a backup:
```
[the remark this answers is not in the document]
Comment (talSofer) [reply docs/howto/garbage-collection/standalone-gc.md]: done

File: docs/howto/garbage-collection/standalone-gc.md [hunk 5]
```diff
@@ -18,67 +18,59 @@ experimental
...
 
-{% include toc_2-4.html %}
+{% include toc_2-3.html %}
 
-## About
+## What is Standalone GC?
 
-Standalone GC is a limited version of the Spark-backed GC that runs without any external dependencies, as a standalone docker image.
+Standalone GC is a simplified version of the Spark-backed GC that runs without any external dependencies, delivered as a standalone
+docker image. It supports S3 and [self-managed S3 compatible storages](#s3-compatible-clients) such as MinIO.
```
Comment (yonipeleg33) [inline docs/howto/garbage-collection/standalone-gc.md:31]: The link is invalid - I think you meant to link the "using-s3-compatible-clients" section?

File: docs/howto/garbage-collection/standalone-gc.md [hunk 6]
```diff
@@ -18,67 +18,59 @@ experimental
...
 
 ## Limitations
 
-1. Except for the [Lab tests](./standalone-gc.md#lab-tests) performed, there are no further guarantees about the performance profile of the Standalone GC. 
-2. Horizontal scale is not supported - Only a single instance of `lakefs-sgc` can operate at a time on a given repository.
-3. Standalone GC only marks objects and does not delete them - Equivalent to the GC's [mark only mode]({% link howto/garbage-collection/gc.md %}#mark-only-mode). \
-   More about that in the [Get the List of Objects Marked for Deletion](./standalone-gc.md#get-the-list-of-objects-marked-for-deletion) section.
-
-### Lab tests
+1. **No horizontal scalability**: Only a single instance of `lakefs-sgc` can operate on a given repository at a time.
```
Comment (yonipeleg33) [inline docs/howto/garbage-collection/standalone-gc.md:35]: Now that I think about it - another limitation compared to the spark-backed GC is that this one only supports AWS (and s3 compatible) storage? the spark-backed one supports any blockstore, right?

File: docs/howto/garbage-collection/standalone-gc.md [hunk 7]
```diff
@@ -136,16 +129,18 @@ The minimal required permissions on LakeFS are:
...
-Currently, `lakefs-sgc` does not provide an option to explicitly set AWS credentials. It relies on the hosting machine
-to be set up correctly, and reads the AWS credentials from the machine.
 
-This means, you should set up your machine however AWS expects you to set it. \
-For example, by following their guide on [configuring the AWS CLI](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html).
+### Credentials
+
+Standalone GC supports S3 and S3-compatible storage backends and relies on AWS credentials for authentication. To set up
+credentials, follow AWS guidelines, such as those outlined in [this guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html).
+For details on how to pass credentials to `lakefs-sgc`, refer to the instructions in [How to Run Standalone GC](#how-to-run-standalone-gc).
```
Comment (yonipeleg33) [inline docs/howto/garbage-collection/standalone-gc.md:137]: Consider being a bit more direct about configuring the docker container:
```suggestion
Standalone GC supports S3 and S3-compatible storage backends and relies on AWS credentials for authentication. To set up
credentials on the `lakefs-sgc` docker container, follow AWS guidelines, such as those outlined in [this guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html).
For details on how to pass credentials to `lakefs-sgc`, refer to the instructions in [How to Run Standalone GC](#how-to-run-standalone-gc).
```

**Review by yonipeleg33 — CHANGES_REQUESTED** [review]
Almost LGTM!
Blocking mainly for the broken link.

File: docs/howto/garbage-collection/standalone-gc.md (hunk 5 above)
Comment (talSofer) [reply docs/howto/garbage-collection/standalone-gc.md]: fixed

File: docs/howto/garbage-collection/standalone-gc.md (hunk 6 above)
Comment (talSofer) [reply docs/howto/garbage-collection/standalone-gc.md:35]: added a bullet

File: docs/howto/garbage-collection/standalone-gc.md (hunk 7 above)
Comment (talSofer) [reply docs/howto/garbage-collection/standalone-gc.md]: done

**Review by yonipeleg33 — APPROVED** [review]
Thank you very much, LGTM!

**Pull request merged**